packages/domain/lib/usecase/analytics_screen_usecase.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:domain/services/analytics_service.dart';
+import 'package:domain/usecase/usecase.dart';
+
+class LogAnalyticsScreenUseCase implements UseCaseParams<String, Future<void>> {
```

Comment:
what is difference between LogAnalyticsEventUseCase and LogAnalyticsScreenUseCase?

Reply:
LogAnalyticsEventUseCase send just event name, LogAnalyticsScreenUseCase send parameter of screen name using one event - screen_view